cinema/tests/test_movie_api.py

```diff
@@ -114,16 +112,12 @@
         }
 
         response = self.client.put(url, payload)
-        self.assertEqual(
-            response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED
-        )
+        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
 
     def test_delete_movie(self):
         movie = sample_movie()
 
         url = detail_url(movie.id)
         response = self.client.delete(url)
 
-        self.assertEqual(
-            response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED
-        )
+        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
```

Comment:
The `test_delete_movie` method expects a `405 Method Not Allowed` response. Please verify that the API is configured to disallow DELETE requests on the movie endpoint if this is the intended behavior. Otherwise, adjust the test to match the actual API behavior.